tools/docker_dev/gitpod.Dockerfile

```
@@ -37,7 +37,7 @@ WORKDIR ${WORKSPACE}
 
 # Build scipy to populate the cache used by ccache
 # Must re-activate conda to ensure the ccache flags are picked up
-RUN git submodule update --init --depth=1 -- scipy/_lib/boost
+RUN git submodule update --init
```

I think we should be specific because it would otherwize pull `sphinxext` and `scipy-mathjax`
```suggestion
RUN git submodule update --init --depth=1 -- scipy/_lib/boost
RUN git submodule update --init --depth=1 -- scipy/_lib/unuran
```

---

Not a gitpod expert: Would that be redundant or lead to a problem? In any case, I will change it this was to keep the scope of the PR minimal.

---

I think this is ok as just running these two commands in docker. @rgommers I will let you merge this one. I am not 100% sure here that's exactly what we want.

---

Either separate or together seems fine, as long as we keep `--depth=1` I think. A full clone can become quite expensive. Probably more so for `boost` than for `unuran`.

---

Thanks Ralf, let's get this in then.